### DISCUSSION TITLE
how can I disable the client side sorting and only enable sever side sorting while using sort prop in ka-table

### DISCUSSION THREAD

**Discussion by SupriyaK06** [komarovalexander/ka-table#392] [Q&A]
I have been trying on sort prop-a handleSortChange function is triggered this function updates the sortfield and sortDirection state which in turn fetches the data from API
but on using this sort prop 1st on clicking the sortable column 1st client side sorting is happening followed but server side sorting
how can I disable the client side sorting and only enable sever side sort.

**Comment by komarovalexander** [comment]
Yes, you can. SingleRemote mode is yours https://komarovalexander.github.io/ka-table/#/sorting-modes

**Reply by SupriyaK06** [reply]
but if having the sorting mode as singleRemote then even the sort prop does not work?
because having the sorting mode as singleRemote its not even triggering the handleSortChange function which is associated with sort prop

**Reply by komarovalexander** [reply]
You can subscribe to onDispatch to handle any table action, demo https://komarovalexander.github.io/ka-table/#/events
